paparazzi/src/main/java/app/cash/paparazzi/ImageSize.kt

```diff
@@ -0,0 +1,8 @@
+package app.cash.paparazzi
+
+import app.cash.paparazzi.internal.ImageUtils
+
+sealed class ImageSize {
+  object FullBleed : ImageSize()
+  data class Limit(val height: Int = ImageUtils.THUMBNAIL_SIZE) : ImageSize()
```

Comment:
`ImageUtils` is internal, as a public user it's not possible to reproduce default behavior without using stable APIs. Maybe simply inline the default height into the constructor usage?

i.e. `private val imageSize: ImageSize = ImageSize.Limit(height = 1000)`